jenkins/slave.sls

```diff
@@ -0,0 +1,60 @@
+{% from "jenkins/map.jinja" import jenkins, deploy with context %}
+
+include:
+  - .sudoersd
+
+jenkins_deps:
```

Comment:
The slave one is meant to be used alone. A jenkins box will never be its own slave.

The jenkins_deps: entry is already present in the master branch.